mail.py

```diff
@@ -47,13 +72,13 @@ def setup_server():
 for uni, toname, seat in students:
     print(uni, msg % seat)
 
-    toaddr = "%s@columbia.edu" % uni
+    toaddr = f"{uni}@columbia.edu"
 
     email = MIMEMultipart()
-    email["From"] = "\"%s\" <%s>" % (fromname, fromaddr)
-    email["To"] = "\"%s\" <%s>" % (toname, toaddr)
-    email["Subject"] = subj
-    email["Reply-To"] = ADDRESS
+    email["From"] = f"\{args.fromname}\{args.fromaddr}"
+    email["To"] = f"\{toname}\{toaddr}"
```

Comment:
I think the format for these two is a little off from the previous one. The string should be something like this:
`"Name" <email-address>`. The quotes should be part of the string (which is why they were escaped in the original), and the angle brackets are necessary as well.